src/table/tables.jl

```diff
@@ -45,17 +45,16 @@ function Tables.schema(table::DTable)
 end
 
 
-function Tables.columnnames(table::DTable)
-    s = Tables.schema(table)
+function determine_columnnames(table::DTable)
+    s = determine_schema(table)
     s === nothing ? nothing : s.names
 end
 
 
 function _getcolumn(table::DTable, col::Union{Symbol, Int})
     chunk_col = (_chunk, _col) -> Tables.getcolumn(_chunk, _col)
     v = [Dagger.spawn(chunk_col, chunk, col) for chunk in table.chunks]
-    p = (_chunk) -> (; :_ => fetch(_chunk))
-    Tables.getcolumn(TableOperations.joinpartitions(Tables.partitioner(p, v)), :_)
+    SentinelArrays.ChainedVector(fetch.(v))
```

Comment:
What's the purpose of using `ChainedVector` here?

Reply:
It's to glue together the column vectors from all the chunks.
To be honest it would be best to use the `DArray` here or some equivalent lazy-loaded column representation, but this is good enough for now I guess